assets/samplesheet.csv

```diff
@@ -1,3 +1,3 @@
 id,fasta,reference,structures
-seatoxin-ref,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/setoxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/setoxin.ref,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/structures/setoxin-ref
-toxin-ref,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/toxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/nf-core-msa/test-dataset/toxin.ref,
+seatoxin-ref,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/setoxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/setoxin.ref,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/structures/setoxin-ref
+toxin-ref,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/toxin-ref.fa,/home/luisasantus/Desktop/crg_cluster/projects/multiplesequencealign/test-dataset/toxin.ref,
```

Comment:
Looks like local paths

Reply:
Ups, yes! fixed it now :)

Reply:
where?

Reply:
nevermind, was looking at the wrong place